jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -141,28 +142,48 @@ def user_api(self):
         return PluginUserApi(
             self,
             expose=(
-                'link_type', 'wcs_use_affine', 'delete_subsets',
-                'viewer', 'orientation',
+                'align_by', 'link_type', 'fast_approximation', 'wcs_use_affine',
+                'delete_subsets', 'viewer', 'orientation',
                 'rotation_angle', 'east_left', 'add_orientation'
             )
         )
 
+    @property
+    @deprecated(since="4.0", alternative="align_by")

```

Comment:
does this work? 🤯 I vaguely recall having troubles getting this to work with properties and setters 🤔  (But assuming it does work, I agree this is much cleaner)

Reply:
It does. `deprecated` has to go after `property` otherwise it complains that the `property` class itself doesn't have an attribute it needs.